extensions/blocks/podcast-player/edit.js

```diff
@@ -124,7 +124,10 @@ const PodcastPlayerEdit = ( {
 					// Show error and allow to edit URL.
 					debug( 'feed error', error );
 					createErrorNotice(
-						__( "Your podcast couldn't be embedded. Please double check your URL.", 'jetpack' )
+						// Error messages already come localized.
+						error.message ||
+							// Fallback to a generic message.
+							__( "Your podcast couldn't be embedded. Please double check your URL.", 'jetpack' )
```

Comment:
this formatting is a tad off to me but that's how the autoformatter has always transformed it

Reply:
It is just because the second statement is long, otherwise it would be onliner. I would probably prefer though to be aligned left 😉